front/app/component-library/components/Icon/Svg.tsx

```diff
@@ -59,9 +69,11 @@ const Svg = ({
       width={width}
       fill={fill}
       transform={transform}
+      aria-labelledby={titleId}
```

Comment:
```suggestion
      // https://www.tpgi.com/using-aria-enhance-svg-accessibility/
      aria-labelledby={titleId}
```

Maybe helpful for future devs?